src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -99,7 +100,7 @@ class DocBaseViewer extends BaseViewer {
         // Call super() to set up common layout
         super.setup();
 
-        this.docEl = this.containerEl.appendChild(document.createElement('div'));
+        this.docEl = this.insertViewerWrapper(document.createElement('div'));
```

Comment:
Could you do the creation in the base setup and just do the assignment here? That would make it more similar to if the div already existed in the shell by the time we got to any specific viewer.

Reply:
Well each viewer has it's own concept of what the viewer wrapper is, whether it is a 'div' or 'pre' or the classes applied to it. Also, it only applies to documents currently so doing it in BaseViewer doesn't seem correct?

Reply:
good point, not worth passing in the Element we want into the super call of setup.